README.md

```diff
@@ -132,12 +153,18 @@ bosdyn-spot-cam-api-msgs             4.0.0
 ```
 If these packages were not installed correctly on your system, you can try manually installing them following [Boston Dynamics' guide](https://dev.bostondynamics.com/docs/python/quickstart#install-spot-python-packages).
 
-## License
+## Numpy & `spot_msgs`
+If `spot_msgs` fails to build because it cannot locate numpy, [this issue](https://github.com/bdaiinstitute/spot_ros2/issues/96) has some solutions to this problem.
+
+## Anaconda
+`spot_ros2` does not currently support a conda install, but it is possible to run the driver in a conda environment with a little extra work -- [this issue](https://github.com/bdaiinstitute/spot_ros2/issues/330) covers some of the additional installs needed for this.
```

Comment:
Can we move the solutions out of those issues? We shouldn't be keeping issues open as documentation. Maybe put them on the wiki for the repo?

Reply:
Spot_ros2 currently doesn't have a wiki page (I know `ros_utilities` does). We can make a ticket for adding one since I think that would be a great place to centralize information. For now I can take these out of the main README.